girder/models/folder.py

```diff
@@ -514,29 +514,42 @@ def parentsToRoot(self, folder, curPath=None, user=None, force=False,
             return self.parentsToRoot(curParentObject, curPath, user=user,
                                       force=force)
 
-    def subtreeCount(self, folder):
+    def subtreeCount(self, folder, includeItems=True, user=None, level=None):
```

Comment:
I think it would make sense for the [user model subtreeCount method](https://github.com/girder/girder/blob/recursive-permission-setting/girder/models/user.py#L286-L299) to expose these additional parameters and pass them on to folder model subtreeCount call.